### ISSUE TITLE
Firebase CLI cannot delete Firestore index exemption

### ISSUE THREAD

**Issue by suztomo** [firebase/firebase-tools#2002]
Creating this issue as per @samtstern  https://groups.google.com/d/msgid/firebase-talk/CAHafJBrtFG8R-dxYh4Jxr0Pk24605fp_YV290S%3DO4cUE9AkOTw%40mail.gmail.com?utm_medium=email&utm_source=footer

>  I wrote that part of the CLI.  At the time I was biasing towards safety since I didn't want someone to run "firebase deploy" and destroy their index configuration. 
> Would you mind filing a feature request on firebase-tools for us to revisit this? 

### [REQUIRED] Environment info

**firebase-tools:**
```
firebase --version                                          git[branch:master]
7.13.1
```
**Platform:**

Mac OS Catalina. 

### [REQUIRED] Test case

Create a index excemption in Firebase Console. Try to delete that via firebase command `firebase deploy` by supplying firestore.indexes.json.

### [REQUIRED] Steps to reproduce

Create a index excemption in Firebase Console. Try to delete that via firebase command `firebase deploy` by supplying firestore.indexes.json.

### [REQUIRED] Expected behavior

Firebase CLI has a way to control Firestore indexes.

### [REQUIRED] Actual behavior

Firebase CLI shows 

```
i  firestore: there are some indexes defined in your project that are not present in your firestore indexes file. Run firebase firestore:indexes and save the result to correct the discrepancy.
i  firestore: there are some field overrides defined in your project that are not present in your firestore indexes file. Run firebase firestore:indexes and save the result to correct the discrepancy.
```

This is an implicit message that the CLI tool cannot delete index exemptions.

**Comment by samtstern** [comment]
@suztomo thanks for filing this!  I was thinking about changing this behavior to ask you (interactively) if you'd like to delete the indexes from your project instead of just ignoring them.  Does that sound good?

**Comment by suztomo** [comment]
Yes, that sounds good.

**Comment by samtstern** [comment]
This will be enabled in the next release (`8.5.0`) of the CLI, see #2327 for implementation.

**Comment by suztomo** [comment]
Thank you.

**Comment by mad-it** [comment]
Im guessing this was a duplicate of https://github.com/firebase/firebase-tools/issues/1127 and should be closed?

**Comment by samtstern** [comment]
@mad-it yes thanks for reminding me!

**Comment by samtstern** [comment]
This was included in `8.4.2` ... we decided this was not a minor version bump but more of a bugfix.
